### DISCUSSION TITLE
Upgrade Deepin base from debian buster to bookworm

### DISCUSSION THREAD

**Discussion by SiamSami** [linuxdeepin/developer-center#6877] [Features Request & Ideas | 特性请求 & 头脑风暴]
The current "stable" distro of debian is codenamed "bookworm". New versions of a lot of packages can't be installed on deepin anymore since it is still based on debian "buster" ("oldoldstable").
It would be really great if the base of Deepin is upgraded to the current stable version of debian namely "bookworm".

**Comment by BLumia** [comment]
deepin v23 no longer based on debian's package repo, and should contain reasonably new packages. (we maintain package by ourself via [the deepin-community GitHub organization](https://github.com/deepin-community/) now).

**Reply by SiamSami** [reply]
Getting glibc version error while trying to launch appimage of xournal++. Says it requires version 2.30

**Reply by raspin0** [reply]
Use flatpak for now
